src/coreclr/vm/excep.h

```diff
@@ -173,30 +173,6 @@ BOOL InstallUnhandledExceptionFilter();
 #endif //!defined(TARGET_UNIX)
 LONG __stdcall COMUnhandledExceptionFilter(EXCEPTION_POINTERS *pExceptionInfo);
 
-
-//////////////
-// A list of places where we might have unhandled exceptions or other serious faults. These can be used as a mask in
-// DbgJITDebuggerLaunchSetting to help control when we decide to ask the user about whether or not to launch a debugger.
-//
-enum UnhandledExceptionLocation
-    {
-    ProcessWideHandler    = 0x000001,
-    ManagedThread         = 0x000002, // Does not terminate the application. CLR swallows the unhandled exception.
```

Comment:
ManagedThread and FinalizerThread were the only two actually used values from this enum.